packages/core/src/libraries/domain.ts

```diff
@@ -104,6 +73,14 @@ export const createDomainLibrary = (queries: Queries) => {
       id: generateStandardId(),
       cloudflareData,
       status: DomainStatus.PendingVerification,
+      dnsRecords: [
+        // Verification CNAME, fixed value, generated by us
+        {
+          type: 'CNAME',
+          name: hostname,
+          value: hostnameProviderConfig.fallbackOrigin,
```

Comment:
just double check this is configured manually or by calling CF API. Manually is good for now, but prefer API if it's possible to reduce config duplication.

Reply:
This can be obtained by an API request, will upgrade it in the next PR.